packages/react/accordion/src/Accordion.tsx

```diff
@@ -29,6 +29,130 @@ const [AccordionContext, useAccordionContext] = createContext<AccordionContextVa
   'Accordion'
 );
 
+/* -------------------------------------------------------------------------------------------------
+ * Accordion
+ * -----------------------------------------------------------------------------------------------*/
```

Comment:
Haha thank the lord 🙏  this has been bugging me for so long 😅

Reply:
haha me too, but everytime I was afraid of muddying the waters 😀